docs/src/content/docs/ko/showcase.mdx

```diff
@@ -115,4 +115,10 @@ import { CardGrid, LinkCard } from '@astrojs/starlight/components';
 		title="starlight-typedoc"
 		description="TypeDoc을 사용하여 TypeScript에서 Starlight 페이지를 생성합니다."
 	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-openapi"
+		title="starlight-openapi"
+		description="Create documentation pages from OpenAPI/Swagger specifications."
```

Comment:
Thanks for the update @jsparkdev!

Looks good, but we should remove the English `description` here:

```suggestion
```